dotcom-rendering/src/web/components/TopicFilterBank.importable.tsx

```diff
@@ -66,6 +66,11 @@ const handleKeyEventClick = (filterKeyEvents: boolean) => {
 	window.location.search = urlParams.toString();
 };
 
+const availableTopicIsSelected =
+	(selectedTopic: Topic) => (availableTopic: Topic) =>
```

Comment:
I'm not sure there's a need to split the arguments like this? Passing them in a single parenthesis would be more idiomatic to DCR / React.

Reply:
agree, might be simpler to read

Reply:
The main reason I did it like this was to be able to use it more easily in the array methods like fineIndex. 

`availableTopics.findIndex(availableTopicIsSelected(selectedTopic));`

Changing it to have 2 params would mean we can't use it without providing the second param straight away and therefore this:
```
const selectedIndex = availableTopics.findIndex((availableTopic) =>
	isEqual(selectedTopic, availableTopic),
);
```

I'm still happy to change if you think it'd make the code simpler. Let me know what you think please :)

Reply:
I think i'd still probably break it down a bit, something like this is quite hard to digest unless you already know what it's doing:
```
availableTopics.findIndex(availableTopicIsSelected(selectedTopic));
```

Reply:
Yea I agree, just changed it 👍

Reply:
Thanks Marjan